pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -126,9 +126,10 @@ def make_pop(input_file: str, random_seed: int = None, time: float = 0):
 
         # Verify all people are logged in cell
         for cell in new_pop.cells:
-            mcell_persons = [person for mcell in cell.microcells
-                             for person in mcell.persons]
-            cell.persons = list(set(cell.persons) | set(mcell_persons))
+            updated_persons = [person for mcell in cell.microcells
+                               for person in mcell.persons]
+            assert len(updated_persons) == len(cell.persons), \
+                "Person gone missing in microcell allocation"
 
```

Comment:
What is a user meant to do about it is this error occurs? Maybe some detail on how to fix